docs/overview/elastic_executor.md

```diff
@@ -0,0 +1,37 @@
+---
+id: overview_elastic_executor
+title: "Elastic Executor"
+---
+
+In order to get the functional effect of executing a specified Elasticsearch request, you must call the `execute` method defined on it, which returns a `ZIO` that requires an `ElasticExecutor`, fails with a `Throwable` and returns the relevant value `A` for that request.
+Elastic requests for creating and deleting return `CreationOutcome` and `DeletionOutcome` respectively if no other meaningful value could be returned, notifying us on the outcome of the request. Any other kind of error is returned as a `Throwable` in the error channel of `ZIO` for that Elastic request.
```

Comment:
`if no other meaningful value could be returned`
Do we need this part? It's not very clear what conditions are we referring to.

Reply:
`Any other kind of error`
We haven't been referring to errors in the previous sentence, so "any other" seems confusing. Perhaps this second sentence is not necessary?